assignments/arrays.js

```diff
@@ -63,32 +63,108 @@ let inventory = [{"id":1,"car_make":"Lincoln","car_model":"Navigator","car_year"
 
 // ==== Challenge 1 ====
 // The dealer can't recall the information for a car with an id of 33 on his lot. Help the dealer find out which car has an id of 33 by logging the car's year, make, and model in the console log provided to you below:
-console.log(`Car 33 is a *car year goes here* *car make goes here* *car model goes here*` );
+
+
+
+var getcarIDinfo = ['2011', 'Jeep', 'Wrangler'];
+    
```

Comment:
you should never use var. For this exercise you could either loop through the array or since you know the id  you could do this
console.log(`Car 33 is a ${inventory[32].car_year} ${inventory[32].car_make} ${inventory[32].car_model}` );